#Inorder to solve the error port already in use run following command
#spring.main.web-application-type=none


#Connection url for the database 
spring.datasource.url = jdbc:mysql://localhost:3306/springboot_securtiy-1?createDatabaseIfNotExist=true&useSSL=false
# Username and password
spring.datasource.username = root
spring.datasource.password = Root@123
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver


#JPA Configurations
spring.jpa.show-sql = true
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
spring.jpa.hibernate.ddl-auto = update

##security default username and password
#spring.security.user.name=root
#spring.security.user.password=root

#logging Properties
logging.level.org.hibernate.SQL=DEBUG
logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE

app.jwtSecretKey ="MINDBOWSER"
app.jwtExpirationsMs =85000000
